sdv/multi_table/base.py

```diff
@@ -107,20 +107,6 @@ def get_table_parameters(self, table_name):
         """
         return self._table_parameters.get(table_name, {})
 
-    def get_parameters(self, table_name):
```

Comment:
There are both a `get_table_parameters` and `get_parameters`. The docs only lists `get_table_parameters` but it actually uses `get_parameters`, so I'm not sure which one we want to keep.

Reply:
what actually uses `get_parameters`?

Reply:
Nothing in SDV. I thought this was the old way to access the parameters before `get_table_parameters` was added, which people forgot to remove. I can double check if any other libraries use it, but my question was more from a user perspective, which method should they use. I think it's `get_table_parameters`, but I wasn't 100% sure because the docs are wrong.

Reply:
They are different methods:
* get_table_parameters(table_name): Get the parameters for a given table name. --> Returns from  `self._table_parameters.get(table_name, {})`
* get_parameters: Returns the parameters that were used to instantiate the model. This should not include the metadata object. Returns from `self._table_synthesizers.get(table_name).get_parameters()`

We may need to file an issue for the docs.

Reply:
That makes sense, thanks. I can file the issue if @amontanez24 is fine with that?